Documentation/ApiOverview/Events/Hooks/Index.rst

```diff
@@ -217,112 +212,47 @@ particular extension.
 $GLOBALS['TYPO3\_CONF\_VARS']['SC\_OPTIONS']
 ============================================
 
-**Configuration space for Core scripts.**
+**Configuration space for Core extensions.**
 
 This array is created as an ad hoc space for creating hooks from any
 script. This will typically be used from the Core scripts of TYPO3
 which do not have a natural identifier like extensions have their
 extension keys.
 
-.. code-block:: php
-   :caption: typo3/sysext/some_extension/ext_localconf.php
+..  literalinclude:: _ext_localconf_schema.php
+    :language: php
+    :caption: EXT:some_extension/ext_localconf.php
 
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['<main_key>']['<sub_key>']['<index>'] = '<function_reference>';
+`<main_key> `
+    The relative path of a script (for output scripts it
+    should be the "script ID" as found in a comment in the HTML header )
 
-- **<main\_key> :** The relative path of a script (for output scripts it
-  should be the "script ID" as found in a comment in the HTML header )
+`<sub_key>`
+    Whatever the script defines. Typically it identifies
+    the context of the hook.
 
-- **<sub\_key> :** Whatever the script defines. Typically it identifies
-  the context of the hook.
+`<index>`
+    Integer index typically. Can be unique string if you have
+    a reason to use that. Normally it has no greater significance since
+    the value of the key is not used. The hooks normally traverse over the
+    array and uses only the value (function reference)
 
-- **<index> :** Integer index typically. Can be unique string if you have
-  a reason to use that. Normally it has no greater significance since
-  the value of the key is not used. The hooks normally traverse over the
-  array and uses only the value (function reference)
+`<function_reference>`
+    A function reference using the syntax of
+    :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::callUserFunction()` as a function
+    or :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()` as a class name
+    depending on implementation of the hook.
 
-- **<function\_reference> :** A function reference using the syntax of
-  :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::callUserFunction()` as a function
-  or :php:`\TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance()` as a class name
-  depending on implementation of the hook.
+    A namespace function has the quoted string format :php:`'Foo\\Bar\\MyClassName->myUserFunction'`
```

Comment:
I think we should only mention the ::class thingy as best practise (can be resolved by IDEs),